docs/tasks/issuers/setup-acme/dns01/route53.rst

```diff
@@ -24,7 +24,10 @@ Cert-manager needs to be able to add records to Route53 in order to solve the DN
            },
            {
                "Effect": "Allow",
-               "Action": "route53:ChangeResourceRecordSets",
+               "Action": [
+                 "route53:ChangeResourceRecordSets",
+                 "route53:ListResourceRecordSets"
+               ]
```

Comment:
```suggestion
               ],
```

Reply:
I think there's a comma missing here for this to be valid.

Reply:
It is indeed, thanks